cli/container_prune.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alibaba/pouch/pkg/utils/filters"
+
+	"github.com/spf13/cobra"
+)
+
+// containerPruneDescription is used to remove all stopped containers.
+var containerPruneDescription = "Remove all stopped containers"
+
+// ContainerPruneCommand use to delete remove all stopped containers.
```

Comment:
rm delete?

Reply:
does rm and delete mean the same things?

Reply:
"delete remove all stopped containers"?  I mean delete is same with remove. So you need to rm either one.